.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
-version: 2
+version: 2.1
 defaults: &defaults
   macos:
-    xcode: 11.7.0
+    xcode: 12.4.0
```

Comment:
If we can, we should update this to the latest stable build of Xcode (12.5.1 at the time I'm writing this)